guide/packages.md

```diff
@@ -60,21 +64,26 @@ The Membrane docker ([Docker Hub](https://hub.docker.com/r/membrane/membrane), [
 | `membrane_flv_plugin`                  | Plugin for muxing audio and video streams to FLV format and demuxing FLV stream to audio and video              | [![Hex.pm](https://img.shields.io/hexpm/v/membrane_flv_plugin.svg)](https://hex.pm/packages/membrane_flv_plugin) [![Docs](https://img.shields.io/badge/api-docs-yellow.svg?style=flat)](https://hexdocs.pm/membrane_flv_plugin) [![GitHub](https://api.iconify.design/octicon:logo-github-16.svg?color=gray&height=20)](https://github.com/membraneframework/membrane_flv_plugin)                                                                     |
 | `membrane_ivf_plugin`                  | Plugin for serializing and deserializng Indeo Video Format stream                                               | [![Hex.pm](https://img.shields.io/hexpm/v/membrane_ivf_plugin.svg)](https://hex.pm/packages/membrane_ivf_plugin) [![Docs](https://img.shields.io/badge/api-docs-yellow.svg?style=flat)](https://hexdocs.pm/membrane_ivf_plugin) [![GitHub](https://api.iconify.design/octicon:logo-github-16.svg?color=gray&height=20)](https://github.com/membraneframework/membrane_ivf_plugin)                                                                     |
+| `membrane_ogg_plugin`                  | [Experimental] Ogg demuxer and parser | [![GitHub](https://api.iconify.design/octicon:logo-github-16.svg?color=gray&height=20)](https://github.com/membraneframework-labs/membrane_ogg_plugin)                                                                     |
+| `membrane_quic_plugin`                  | [Experimental] Plugin containing elements for sending and receiving data over QUIC. It contains also QUIC server | [![GitHub](https://api.iconify.design/octicon:logo-github-16.svg?color=gray&height=20)](https://github.com/mickel8/membrane_quic_plugin)                                                                     |
+| `membrane_hls_plugin`                  | [Third-Party] [Alpha] Membrane.HLS.Source element for HTTP Live Streaming (HLS) playlist files                                                                   | [![GitHub](https://api.iconify.design/octicon:logo-github-16.svg?color=gray&height=20)](https://github.com/kim-company/membrane_hls_plugin)                                                                     |
+| `membrane_ogg_plugin`                  | [Third-Party] [Alpha] Ogg plugin based on the libogg                                                                  | [![GitHub](https://api.iconify.design/octicon:logo-github-16.svg?color=gray&height=20)](https://github.com/ledhed2222/membrane_ogg_plugin)                                                                     |
```

Comment:
The name is the same as with our `membrane_ogg_plugin` - can it be left that way?

Reply:
I suppose - none of them is released.

Reply:
One of them is `membrane_libgg_plugin`

Reply:
https://github.com/membraneframework-labs/membrane_ogg_plugin/blob/graszka22/ogg_depayload/mix.exs
https://github.com/ledhed2222/membrane_ogg_plugin/blob/main/mix.exs
Which one?